packages/wasm/wasm.0.1/url

```diff
@@ -0,0 +1,2 @@
+archive: "https://github.com/WebAssembly/spec/archive/0xd.tar.gz"
```

Comment:
If it's ok to have hexadecimal version numbers in the package path, then sure, happy to change -- I just assumed that OPAM had no support for this.